serialization.md

```diff
@@ -26,13 +26,6 @@ This specification uses the following terms:
 	Changing it means creating a new derived image, instead of changing the existing image.
     </dd>
     <dt>
-        Image Filesystem Changeset
-    </dt>
-    <dd>
-        Each layer has an archive of the files which have been added, changed, or deleted relative to its parent layer.
-	Using a layer-based or union filesystem such as AUFS, or by computing the diff from filesystem snapshots, the filesystem changeset can be used to present a series of image layers as if they were one cohesive filesystem.
```

Comment:
It is okay to repeat some information to enhance context. Please merge these descriptions.